cmd/project.go

```diff
@@ -804,6 +806,81 @@ Examples:
 	},
 }
 
+var projectRollbackDeploymentCmd = &cobra.Command{
+	Use:   "rollback-deployment",
+	Short: "Rollback a deployment for a project",
+	Long: `Rollback a specific deployment for a project by selecting a project and deployment.
+
+Examples:
+  agentuity project rollback-deployment
+`,
+	Args: cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := env.NewLogger(cmd)
+		ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
+		defer cancel()
+		apikey, _ := util.EnsureLoggedIn(ctx, logger, cmd)
+		apiUrl, _, _ := util.GetURLs(logger)
+
+		selected := selectProject(ctx, logger, apiUrl, apikey, "Select a project to rollback a deployment")
```

Comment:
we should be consistent with deploy and require a directory flag or be in the current directory.